.github/workflows/ci-job.yml

```diff
@@ -125,3 +125,4 @@ jobs:
       GH_ACTIONS: 1
       RUST_BACKTRACE: full
       RUSTFLAGS: -D warnings
+      V8_FORCE_DEBUG: 1
```

Comment:
Revert before merging? Or maybe add another CI step that runs in debug mode only on linux?